src/UI/Buyer/src/app/shared/services/oc-reorder/oc-reorder.service.ts

```diff
@@ -0,0 +1,113 @@
+import { Injectable } from '@angular/core';
+import { AppLineItemService } from '@app-buyer/shared/services/oc-line-item/oc-line-item.service';
+import { orderReorderResponse } from '@app-buyer/shared/services/oc-reorder/oc-reorder.interface';
+import { Observable, of, forkJoin } from 'rxjs';
+import { flatMap } from 'rxjs/operators';
+import { OcMeService, BuyerProduct, LineItem } from '@ordercloud/angular-sdk';
+import {
+  map as _map,
+  forEach as _forEach,
+  find as _find,
+  differenceBy as _differenceBy,
+} from 'lodash';
+
+@Injectable()
```

Comment:
For this service, lets mark functions not used outside as 'private'. Makes it a little easier to see the entry points.